core/test/computed/metrics/interactive-test.js

```diff
@@ -26,16 +40,20 @@ function generateNetworkRecords(records, timeOrigin) {
       startTime: (item.start + timeOriginInMs) / 1000,
       endTime: item.end === -1 ? -1 : (item.end + timeOriginInMs) / 1000,
     };
+    return /** @type {LH.Artifacts.NetworkRequest} */ (record);
```

Comment:
seems like it would be better to move to network-records-to-devtools-log to get a real network request rather than coercing a fake?